### PR TITLE
[minor][cdc][docs] Optimize styles of the Flink CDC website docs home…

### PR DESCRIPTION
**Pull request by Laffery**
Optimize styles of the Flink CDC website documentation homepage:

- support responsive styles to improve reading experience under mobile device.
- optimize the background image to avoid large blank areas and increase the information content of the first screen
- highlight the`Quick Start` button to help users find it well

Take iPhone 14pm (430px, 932px) for an example 

**Before:**
image
image
image

**After:**
image
image
image

### CODE REVIEW

**Comment by leonardBang** [comment]
@GOODBOY008 Would you like to help review this PR？

**Review by GOODBOY008 — CHANGES_REQUESTED** [review]
@Laffery Please add `/` before `fig`.

**Comment by Laffery** [comment]
> @Laffery Please add `/` before `fig`. And ci will pass.

Thanks for your kind tips, I've finished and hope you review and trigger the CI again 🙏

File: docs/content.zh/_index.md [hunk 1]
```diff
@@ -22,177 +22,163 @@ specific language governing permissions and limitations
...
+            <div class="w-full w-4/5 flex text-center">
+                <div class="basis-1/12"></div>
+                <p class="text-lg basis-10/12">
+                    Flink CDC is a distributed data integration tool for real time data and batch data. Flink CDC brings the simplicity and elegance of data integration via YAML to describe the data movement and transformation.
+                </p>
+                <div class="basis-1/12"></div>
+            </div>
+            <div class="w-full flex justify-center	">
+                <div class="w-4/5">
+                    {{<img src="/fig/index-yaml-example.png" alt="Flink CDC Example">}}
```
Comment (GOODBOY008) [inline docs/content.zh/_index.md:74]:
```suggestion
                    {{< img src="/fig/index-yaml-example.png" alt="Flink CDC Example" >}}
```

File: docs/content.zh/_index.md [hunk 2]
```diff
@@ -22,177 +22,163 @@ specific language governing permissions and limitations
...
+            </h1>
+            <p class="leading-normal text-xl text-white">
+                A streaming data integration tool
+            </p>
+            <a href="{{< ref "docs/get-started/introduction" >}}" class="link-as-button mx-auto bg-white text-black font-bold my-4 px-8">
+                Quick Start
+            </a>
+        </div>
+        <div class="w-full md:w-3/5 py-6 text-center">
+            {{<img src="/fig/cdc-flow.png" alt="Flink CDC Flow">}}
```
Comment (GOODBOY008) [inline docs/content.zh/_index.md:39]:
```suggestion
            {{< img src="/fig/cdc-flow.png" alt="Flink CDC Flow" >}}
```

**Review by GOODBOY008 — CHANGES_REQUESTED** [review]
@Laffery I left some suggestions.

**Comment by Laffery** [comment]
> @Laffery I left some suggestions.

Ohh it's really helpful on these details, I've commited again according to your suggestions.

**Review by leonardBang — APPROVED** [review]
Thanks @Laffery for the contribution and @GOODBOY008 for the review, LGTM

**Pull request merged**